stats/memory_reporter.go

```diff
@@ -75,7 +75,10 @@ func (m *MemoryReporter) WriteGraphiteLine(buf, prefix []byte, now time.Time) []
 	}
 
 	// metric memory.gc.gogc is the current GOGC value (derived from the GOGC environment variable)
-	buf = WriteInt32(buf, prefix, []byte("memory.gc.gogc.sgauge32"), nil, nil, int32(gcPercent), now)
+	buf = WriteInt32(buf, prefix, []byte("memory.gc.gogc.gauge32"), nil, nil, int32(gcPercent), now)
+
+	// metric runtime.goroutines.total is how many goroutines there are
+	buf = WriteUint64(buf, prefix, []byte("runtime.goroutines.total.gauge64"), nil, nil, m.mem.HeapObjects, now)
```

Comment:
Did you want to use something like `uint64(runtime.NumGoroutine())` instead of `m.mem.HeapObjects` here?

Reply:
d'oh! that explained why the chart looked strange in my testing :x

Reply:
fixed